src/test/java/hudson/plugins/git/browser/BitbucketServerTest.java

```diff
@@ -0,0 +1,99 @@
+/**
+ * Copyright 2010 Mirko Friedenhagen
+ */
+
+package hudson.plugins.git.browser;
+
+import hudson.EnvVars;
+import hudson.model.TaskListener;
+import hudson.plugins.git.GitChangeLogParser;
+import hudson.plugins.git.GitChangeSet;
+import hudson.plugins.git.GitChangeSet.Path;
+import org.jenkinsci.plugins.gitclient.Git;
+import org.jenkinsci.plugins.gitclient.GitClient;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URL;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Random;
+import static org.junit.Assert.*;
+import org.junit.Test;
+
+/**
+ * @author mattsemar
```

Comment:
I assume that this is the author of the original tests, not the author of these tests?

```suggestion
```